apps/system/js/notifications.js

```diff
@@ -468,9 +468,11 @@ var NotificationScreen = {
 
     notificationNode.appendChild(titleContainer);
 
+    var messageWrapper = document.createElement('bdi');
```

Comment:
you don't need <bdi> element since you set dir="auto" on the toaster-detail it acts as a bdi.